internal/replication/blockreplicator_reconfig_test.go

```diff
@@ -91,19 +110,14 @@ func TestBlockReplicator_ReConfig_Endpoint(t *testing.T) {
 
 		// submission to a follower will cause an error
 		err = env.nodes[follower1].blockReplicator.Submit(b)
-		assert.EqualError(t, err, expectedNotLeaderErr)
-		// submission to a partitioned node will cause an error
-		err = env.nodes[follower2].blockReplicator.Submit(b)
-		assert.EqualError(t, err, "not a leader, leader is RaftID: 0, with HostPort: ")
+		require.EqualError(t, err, expectedNotLeaderErr)
```

Comment:
`follower2` updated couple of lines above, but not used anymore

Reply:
Done